include/libtorrent/udp_socket.hpp

```diff
@@ -100,6 +100,13 @@ namespace libtorrent
 		bool is_closed() const { return m_abort; }
 		udp::endpoint local_endpoint(error_code& ec) const
 		{ return m_socket.local_endpoint(ec); }
+		// best effort, if you want to know the error, use
+		// ``local_endpoint(error_code& ec)``
+		udp::endpoint local_endpoint() const
```

Comment:
well, I think it improve the code in some places? do you think it's dangerous to have it?